recipes/samarium/all/conandata.yml

```diff
@@ -8,6 +8,21 @@ sources:
 patches:
   "1.0.0":
     - patch_file: "patches/1.0.0-updated-cmake.patch"
+      patch_description: "Use find_package for CMakeDeps"
+      patch_type: "fix"
```

Comment:
```suggestion
      patch_type: "conan"
```

Reply:
Sorry, missed seeing the valid patch types